tests/test_config.py

```diff
@@ -386,3 +386,56 @@ def test_umask_config(options, expected):
     with AltArgs(cmdline):
         app = NoConfigApp()
     assert app.cfg.umask == expected
+
+
+@pytest.mark.skipif(sys.version_info < (3, 0) or sys.version_info > (3, 6),
+    reason="requires python3.0-3.6")
+@pytest.mark.parametrize("options, expected", [
+    (["--ssl-version", "SSLv3"], 1),
+    (["--ssl-version", "1"], 1),
+])
+def test_ssl_version_named_constants_python_30_to_36(options, expected):
+    _test_ssl_version(options, expected)
+
+
+@pytest.mark.skipif(sys.version_info < (3, 0),
```

Comment:
master branch no longer supports Python 2, so `skipif` can be removed.

Reply:
Done, removed.